src/IssueViz.UnitTests/Selection/IssueFlowStepNavigatorTests.cs

```diff
@@ -314,10 +270,17 @@ private IAnalysisIssueLocationVisualization CreateLocation(int stepNumber = 1, b
             var locationViz = new Mock<IAnalysisIssueLocationVisualization>();
             locationViz.Setup(x => x.Location).Returns(Mock.Of<IAnalysisIssueLocation>());
             locationViz.Setup(x => x.StepNumber).Returns(stepNumber);
-            locationViz.SetupProperty(x => x.IsNavigable);
-            locationViz.Object.IsNavigable = isMarkedAsNavigable;
 
-            locationNavigatorMock.Setup(x => x.TryNavigate(locationViz.Object.Location)).Returns(isTrulyNavigable);
+            if (isMarkedAsNavigable)
```

Comment:
This name is no longer correct but I can't think of a better one. `isPotentiallyNavigable`?

Reply:
I think either is ok.